<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>964</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>880</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>871</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="meatTab">
    <attribute name="title">
     <string>Meat</string>
    </attribute>
    <widget class="QTableWidget" name="meatTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>851</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="vegeTab">
    <attribute name="title">
     <string>Vegetable</string>
    </attribute>
    <widget class="QTableWidget" name="vegeTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>851</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="soupTab">
    <attribute name="title">
     <string>Soup</string>
    </attribute>
    <widget class="QTableWidget" name="soupTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>851</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stapleTab">
    <attribute name="title">
     <string>Staple</string>
    </attribute>
    <widget class="QTableWidget" name="stapleTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>851</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="snackTab">
    <attribute name="title">
     <string>Snack</string>
    </attribute>
    <widget class="QTableWidget" name="snackTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>851</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="orderTab">
    <attribute name="title">
     <string>提交菜单</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>桌号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="tableSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>人数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="hcSpinBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="checkButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>790</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确认提交</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="orderTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>531</width>
       <height>691</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>890</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
